src/sqlalchemy_cratedb/dialect.py

```diff
@@ -377,6 +398,17 @@ def has_ilike_operator(self):
         server_version_info = self.server_version_info
         return server_version_info is not None and server_version_info >= (4, 1, 0)
 
+    def _format_query(self, query):
+        """
+        When using the PostgreSQL protocol with drivers `psycopg` or `asyncpg`,
+        the paramstyle is not `qmark`, but `pyformat`.
```

Comment:
I am not quite following. I assume we would want to use server side binding (i.e. qmark)?

Reply:
This is probably the reason for the comment and the workaround: Because PostgreSQL drivers psycopg and asyncpg use `pyformat`, but CrateDB uses `qmark`, we may need to adjust.